src/vega.schema.ts

```diff
@@ -320,7 +320,14 @@ export type VgEncodeChannel =
   | 'href'
   | 'cursor'
   | 'defined'
-  | 'cornerRadius';
+  | 'cornerRadius'
+  | 'cornerRadiusTopLeft'
+  | 'cornerRadiusTopRight'
+  | 'cornerRadiusBottomRight'
+  | 'cornerRadiusBottomLeft'
+  | 'scaleX'
```

Comment:
Are scaleX and scaleY related to this PR? or this file is from Vega?

Reply:
I noticed that they are missing. Not related to this pull request.